pcsx2/GS/Renderers/HW/GSRendererHW.cpp

```diff
@@ -1543,12 +1585,14 @@ void GSRendererHW::Draw()
 		else if ((m_context->CLAMP.WMT & 2) && !(tmm.uses_boundary & TextureMinMaxResult::USES_BOUNDARY_V))
 			m_context->CLAMP.WMT = CLAMP_CLAMP;
 #endif
-
+		const int tw = 1 << TEX0.TW;
+		const int th = 1 << TEX0.TH;
+		const bool is_shuffle = m_channel_shuffle | m_texture_shuffle;
```

Comment:
Should use logical ||.

Reply:
Why? both are boolean, it doesn't matter.

Reply:
VS complains about it.